plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php

```diff
@@ -219,6 +222,45 @@ public function add_unit_price( $item_id, $item, $order ) {
 			echo wp_kses_post( '<br /><small>' . $unit_price . '</small>' );
 		}
 
+		/**
+		 * Add additional details to the order item totals table.
+		 *
+		 * @param array    $total_rows Array of total rows.
+		 * @param WC_Order $order      Order object.
+		 * @param string   $tax_display Tax display.
+		 * @return array Modified array of total rows.
+		 */
+		public function order_item_totals( $total_rows, $order, $tax_display ) {
+			$cash_payment_change_due_amount           = $order->get_meta( '_cash_change_amount', true );
+			$formatted_cash_payment_change_due_amount = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
+			if ( null !== $cash_payment_change_due_amount && '' !== $cash_payment_change_due_amount ) {
```

Comment:
Optional, but we could use PHP's `empty()` to replace both of these checks and other similar checks in this code.